apps/SageAccountWeb/src/App.tsx

```diff
@@ -28,70 +24,65 @@ import { getSearchParam } from 'URLUtils'
 import './App.scss'
 import AppInitializer from './AppInitializer'
 import LoginPage from './LoginPage'
-import generalTheme from './style/theme'
 import { QueryClient, QueryClientProvider } from 'react-query'
+import ApplicationSessionManager from 'synapse-react-client/dist/utils/apputils/session/ApplicationSessionManager'
 
 const isCodeSearchParam = getSearchParam('code') !== undefined
 const isProviderSearchParam = getSearchParam('provider') !== undefined
 const isInSSOFlow = isCodeSearchParam && isProviderSearchParam
 
-// theme is a merge of a general theme and particular color pallettesfor the source app
-const theme = createTheme(generalTheme)
-
-const App: React.FC = () => {
-  const queryClient = new QueryClient({
-    defaultOptions: {
-      queries: {
-        staleTime: 50 * 1000, // 50s
-        retry: false, // SynapseClient knows which queries to retry
-      },
+const queryClient = new QueryClient({
+  defaultOptions: {
+    queries: {
+      staleTime: 50 * 1000, // 50s
+      retry: false, // SynapseClient knows which queries to retry
     },
-  })
+  },
+})
 
+function App() {
   return (
     <div className="App">
-      <StyledEngineProvider injectFirst>
-        <ThemeProvider theme={theme}>
-          <Router>
-            <QueryClientProvider client={queryClient}>
+      <Router>
+        <QueryClientProvider client={queryClient}>
+          <StyledEngineProvider injectFirst>
+            <ApplicationSessionManager>
               <AppInitializer>
                 <CookiesNotification />
                 <Switch>
-                  <Route
-                    exact
-                    path="/"
-                    render={props => {
-                      return (
-                        <SynapseContextConsumer>
-                          {(ctx?: SynapseContextType) => {
-                            if (!ctx?.accessToken) {
-                              return <LoginPage returnToUrl={'/'} />
-                            } else {
-                              return (
-                                <AppContextConsumer>
-                                  {appContext => (
-                                    <>
-                                      {appContext?.redirectURL &&
-                                        !isInSSOFlow &&
-                                        window.location.replace(
-                                          appContext?.redirectURL,
-                                        )}
-                                    </>
-                                  )}
-                                </AppContextConsumer>
-                              )
-                            }
-                          }}
-                        </SynapseContextConsumer>
-                      )
-                    }}
-                  />
+                  <Route exact path="/">
```

Comment:
In general I prefer 
```tsx
<Route>
  <Component />
</Route>
```

over 

```tsx
<Route 
  render={() => {
    // ...
  }}
/>
```

The latter makes it easier to write really complex inline components, which ends up becoming harder and harder to understand.

There is a difference between the two. `children` will always be rendered, but hidden if the path doesn't match. `render` is only rendered if the path matches. So it does matter for those routes with side effects (e.g. the logout route)